src/pages/HomePage/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,33 @@
+interface IRatingProps {
+    zeroData: string;
+    firstData: string;
+    secondData: string;
+    thirdData: string;
+    forthData: string;
+    fifthData: string;
+    star: string;
+  }
+  
+  export const Rating = (props: IRatingProps) => {
+    const { zeroData, firstData, secondData, thirdData, forthData, fifthData, star } = props;
+    return (
+      <div className="rating" data-total-value={zeroData}>
+        <div className="stars" data-item-value={fifthData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={forthData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={thirdData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={secondData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={firstData}>
+          {star}
+        </div>
```

Comment:
I'm pretty much sure that I left such comment somewhere...
you component must have a number (rating's value) as an input parameter. and it must have logic to draw as many yellow stars as it is needed) the same for gray stars

oh, and you must merge develop into you branch (just too many conflicts)